src/containers/Header/breadcrumbs.tsx

```diff
@@ -145,23 +145,27 @@ const getVDiskBreadcrumbs: GetBreadcrumbs<VDiskBreadcrumbsOptions> = (options, q
 };
 
 const getTabletsBreadcrumbs: GetBreadcrumbs<TabletsBreadcrumbsOptions> = (options, query = {}) => {
-    const {tenantName, nodeIds} = options;
+    const {tenantName, nodeId} = options;
 
     const tenantQuery = getQueryForTenant('tablets');
 
     const breadcrumbs = tenantName
         ? getTenantBreadcrumbs(options, {...query, ...tenantQuery})
         : getClusterBreadcrumbs(options, query);
 
-    const link = createHref(routes.tabletsFilters, undefined, {
-        ...query,
-        nodeIds,
-        path: tenantName,
-    });
-
-    const lastItem = {text: headerKeyset('breadcrumbs.tablets'), link};
-    breadcrumbs.push(lastItem);
-
+    if (nodeId) {
+        const link = createHref(
+            routes.node,
+            {id: nodeId, activeTab: TABLETS},
+            {
+                ...query,
+                tenantName,
+            },
+        );
+
+        const lastItem = {text: headerKeyset('breadcrumbs.tablets'), link};
+        breadcrumbs.push(lastItem);
```

Comment:
Why did not you use "Node" breadcrumb instead? Currently, we have "Tablets" breadcrumb, which is changed to "Node NodeId", when I click